CMakeLists.txt

```diff
@@ -37,6 +37,18 @@ option(MLN_QT_WITH_WIDGETS "Build QMapLibreWidgets" ON)
 option(MLN_QT_STATIC "Build QMapLibre staticaly (force static build with Qt6)" OFF)
 option(MLN_QT_WITH_COVERAGE "Build QMapLibre with code coverage collection" OFF)
 option(MLN_QT_WITH_CLANG_TIDY "Build QMapLibre with clang-tidy checks enabled" OFF)
+option(MLN_QT_WITH_LTO "Build QMapLibre with Link-Time Optimization" OFF)
+
+if(MLN_QT_WITH_LTO)
+    include(CheckIPOSupported)
+    check_ipo_supported(RESULT ipo_result OUTPUT ipo_output)
```

Comment:
Is this check really needed?

I think CMake handles only setting it when it is supported.
https://cmake.org/cmake/help/latest/variable/CMAKE_INTERPROCEDURAL_OPTIMIZATION.html
https://cmake.org/cmake/help/latest/prop_tgt/INTERPROCEDURAL_OPTIMIZATION.html#prop_tgt:INTERPROCEDURAL_OPTIMIZATION

Reply:
I thought it may be useful to warn people if it wouldn't work.